### PR TITLE
Replace dead inqstats api by population.io api

### PR DESCRIPTION
**Pull request by TheRizzen**
Inqstats link in the OpenData section seems to be dead.
I'm currently using api.population.io on one of my projects and it seems to offer quite similar data.
Thus, I just replaced the Inqstats entry.

### CODE REVIEW

**Comment by davemachado** [comment]
Hi @TheRizzen , thanks for the PR. Please remove any merge conflicts.

**Comment by TheRizzen** [comment]
It's done. Merge conflict must have been introduced by previous merges.

File: README.md [hunk 1]
```diff
@@ -517,10 +517,10 @@ API | Description | Auth | HTTPS | CORS | Link |
 | Enigma Public | Broadest collection of public data | `apiKey` | Yes | Yes | [Go!](http://docs.enigma.com/public/public_v20_api_about) |
 | fonoApi | Mobile Device Description | No | Yes | Unknown | [Go!](https://fonoapi.freshpixl.com/) |
 | French Address Search | Address search via the French Government | No | Yes | Unknown | [Go!](https://adresse.data.gouv.fr/api) |
-| INQStats | Open demographic data such as population, life expectancy, migration rate, etc | `apiKey` | No | Unknown | [Go!](http://blog.inqubu.com/inqstats-open-api-published-to-get-demographic-data) |
 | LinkPreview | Get JSON formatted summary with title, description and preview image for any requested URL | `apiKey` | Yes | Yes | [Go!](https://www.linkpreview.net) |
 | Marijuana Strains | Marijuana strains, races, flavors, and effects | `apiKey` | No | Unknown | [Go!](http://strains.evanbusse.com/) |
-| Microlink.io | Extract structured data from any website | No | Yes | Yes | [Go!](https://microlink.io) |
+| Microlink.io | Turns any link into information | No | Yes | Unknown | [Go!](https://docs.microlink.io) |
```
Comment (davemachado) [inline README.md]: I don't think you meant to change this line?

Comment (TheRizzen) [reply README.md]: My bad. Kept the version of the line before the commit. It's fixed.

**Comment by davemachado** [comment]
Thanks!

**Pull request merged**